### ISSUE TITLE
App updates offered on google play before f-droid

### ISSUE THREAD

**Issue by Oclair** [ProtonVPN/android-app#135]
Why do updates hit play store before f-droid repos?

Good has a rude behavior attempting to update apps from other app stores.

Maybe expedite updates pushed to f-droid

Thanks l

**Comment by Mr-Hitchens** [comment]
I agree. Every other app that uses both stores updates almost immediately. Proton not so much. Why?

**Comment by mateusz-markowicz** [comment]
F-droid is supposed to pick-up new revision once we push to github, 5.3.15.3 was pushed on 2.05. We reported the issue here: https://gitlab.com/fdroid/fdroiddata/-/issues/3261

**Comment by mateusz-markowicz** [comment]
Newest version available on https://f-droid.org/packages/ch.protonvpn.android/

**Comment by Gitchick57** [comment]
What is the difference between updating from GitHub vs. F-droid?



May 27, 2024, 6:05 AM by ***@***.***:

**Comment by Oclair** [comment]
Issue re appeared

**Comment by Gitchick57** [comment]
Also on GitHub.  The last update was in April, I believe.



May 31, 2024, 8:23 AM by ***@***.***:

**Comment by Oclair** [comment]
bump...

**Comment by licaon-kter** [comment]
the team forgot to update the version they want F-Droid to build: https://github.com/ProtonVPN/android-app/blob/5.6.8.0/metadata/version_fdroid.txt or maybe they decided to not let F-Droid build it... .they would know

so @Oclair fyi at the moment F-Droid has the "latest" version

**Comment by CaptechOmar** [comment]
Again, there is an update on playstore which I can't install
And I can't install from fdroid as there is no update

**Comment by licaon-kter** [comment]
Was built (https://monitor.f-droid.org/builds/log/ch.protonvpn.android/605112901#site-footer) awaits publishing.

Read https://gitlab.com/fdroid/wiki/-/wikis/FAQ#how-long-does-it-take-for-my-app-to-show-up-on-website-and-client while you wait
